containers/src/impl/Kokkos_Functional_impl.hpp

```diff
@@ -105,16 +105,18 @@ uint32_t MurmurHash3_x86_32(const void* key, int len, uint32_t seed) {
 
   uint32_t k1 = 0;
 
-  switch (len & 3) {
-    case 3: k1 ^= tail[2] << 16;
-    case 2: k1 ^= tail[1] << 8;
-    case 1:
-      k1 ^= tail[0];
-      k1 *= c1;
-      k1 = rotl32(k1, 15);
-      k1 *= c2;
-      h1 ^= k1;
-  };
+  const int len_and_3 = len & 3;
```

Comment:
uhm is this the same thing? for example `k1^=tail[1] << 8` is now executed when `len&3==3` which wasn't true before...

Reply:
AFAICT, it was supposed to fall through.